dspace-api/src/main/java/org/dspace/app/mediafilter/MediaFilterServiceImpl.java

```diff
@@ -352,10 +353,20 @@ public boolean processBitstream(Context context, Item item, Bitstream source, Fo
                 + " (item: " + item.getHandle() + ")");
         }
 
-        InputStream destStream;
+        InputStream destStream = null;
         try {
             System.out.println("File: " + newName);
-            destStream = formatFilter.getDestinationStream(item, bitstreamService.retrieve(context, source), isVerbose);
+            InputStream srcStream = bitstreamService.retrieve(context, source);
+            destStream = formatFilter.getDestinationStream(item, srcStream, isVerbose);
+            if (srcStream != null)
+            {
+                try {
+                    srcStream.close();
+                } catch (IOException ex)
+                {
+                    // nothing to be done here.
+                }
+            }
```

Comment:
The above if-block can be replaced by org.apache.commons.io.IOUtils#closeQuietly(java.io.InputStream)

Reply:
Try-with-resources might also work, and is even less noisy.

Reply:
That's correct since we can assume Java 7 now.